Userland/Services/Taskbar/ClockWidget.cpp

```diff
@@ -196,6 +199,24 @@ void ClockWidget::mousedown_event(GUI::MouseEvent& event)
     }
 }
 
+void ClockWidget::context_menu_event(GUI::ContextMenuEvent& event)
+{
+    if (!m_context_menu) {
+        m_context_menu = GUI::Menu::construct();
+
+        auto settings_icon = MUST(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/settings.png"sv));
+        auto open_clock_settings_action = GUI::Action::create("Clock &Settings", *settings_icon, [](auto&) {
+            auto result = Core::Process::spawn("/bin/ClockSettings"sv, Array { "--open-tab"sv, "clock"sv });
```

Comment:
Tiny nit: It's better not to use StringViews for the argument array, if possible, since `::spawn()` then has to convert them to Strings to guarantee null-termination. I.e. just don't add `sv`.
```suggestion
            auto result = Core::Process::spawn("/bin/ClockSettings"sv, Array { "--open-tab", "clock" });
```
(Also maybe see `GUI::Process::spawn_or_show_error`).

Reply:
Ah that's unfortunate, I forgot `spawn` args needs to be `NUL`-terminated. Probably not worth having the `StringView` overload :thinking: but that's for another day.